geniza/corpus/models.py

```diff
@@ -768,9 +768,13 @@ def iiif_images(self, filter_side=False, with_placeholders=False):
                             and i not in b.selected_images,
                         }
 
-        # when requested, include any placeholder canvas URIs referenced by any associated transcriptions
+        # when requested, include any placeholder canvas URIs referenced by any associated
+        # transcriptions or translations
         if with_placeholders:
-            for ed in self.digital_editions().all():
+            for ed in [
+                *self.digital_editions().all(),
+                *self.digital_translations().all(),
```

Comment:
this will be two separate db calls, couldn't we get what we want in one query?

Reply:
probably could write a queryset filter that would give distinct canvas uris across all annotations for this document